junit-platform-commons/src/main/java/org/junit/platform/commons/support/conversion/ConversionSupport.java

```diff
@@ -98,49 +49,52 @@ private ConversionSupport() {
 	 * type is a reference type
 	 *
 	 * @since 1.11
+	 * @see DefaultConverter
+	 * @deprecated Use {@link #convert(Object, TypeDescriptor, ClassLoader)} instead.
+	 */
+	@Deprecated
+	@API(status = DEPRECATED, since = "5.13")
+	public static <T> @Nullable T convert(@Nullable String source, Class<T> targetType, ClassLoader classLoader) {
+		return convert(source, TypeDescriptor.forClass(targetType), getClassLoader(classLoader));
+	}
+
+	/**
+	 * Convert the supplied source object into an instance of the specified
+	 * target type.
+	 *
+	 * @param source the source object to convert; may be {@code null}
+	 * but only if the target type is a reference type
+	 * @param targetType the target type the source should be converted into;
+	 * never {@code null}
+	 * @param classLoader the {@code ClassLoader} to use; may be {@code null} to
+	 * use the default {@code ClassLoader}
+	 * @param <T> the type of the target
+	 * @return the converted object; may be {@code null} but only if the target
+	 * type is a reference type
+	 *
+	 * @since 1.13
 	 */
+	@API(status = EXPERIMENTAL, since = "1.13")
```

Comment:
If I remember correctly, the platform will jump to version 6, right?

```suggestion
	@API(status = EXPERIMENTAL, since = "6.0")
```

Reply:
Yep, that's correct. :+1: